src/data/faq_de.json

```diff
@@ -376,6 +376,217 @@
+                                "anchor": "warn_without_tan_pcr_after_self_test",
+                                "active": false,
+                                "textblock": [
+                                    "Aus epidemiologischer Sicht ist ein PCR-Test natürlich deutlich genauer und aussagekräftiger als ein von Laien durchgeführter Selbsttest. Daher wird zur Bestätigung (bzw. zur Überprüfung von ggf. False-Positives) des Selbsttests ein PCR-Test empfohlen.",
+                                    "Um die lokalen Bestimmungen zu befolgen, empfehlen wir daher, dass sie nach einem positiven Selbsttest Kontakt mit dem zuständigen Gesundheitsamt oder dem Arzt Ihres Vertrauens aufnehmen und das weitere Vorgehen besprechen."
+                                ]
+                            },
+                            {
+                                "title": "Was passiert, wenn ich nach einem positiven Selbsttest noch einmal mit einem positiven PCR-Test warne? Geht das überhaupt?",
+                                "anchor": "warn_without_tan_warn_twice",
+                                "active": false,
+                                "textblock": [
+                                    "Die neue Funktion lässt nach einem erfassten Selbsttest nur noch einen weiteren registrierten PCR-/Schnelltest-Test zu. Falls mit dem positiven Selbsttest schon gewarnt wurde, würden durch einen solchen weiteren „offiziellen“ Test mit positivem Ergebnis dann ggf. einige der Kontakte eine weitere Risikowarnung erhalten.",
+                                    "Wir empfehlen aber weiterhin, sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig zu verhalten, dass keine anderen Personen angesteckt werden. Dann sind weitere Warnungen auch nicht mehr nötig."
+                                ]
+                            },
+                            {
+                                "title": "Einige Tage nach dem letzten positiven Test bin ich beim Selbsttest noch immer positiv. Soll ich nochmal warnen? Warum funktioniert das beim zweiten Mal nicht?",
+                                "anchor": "warn_without_tan_still_positive",
+                                "active": false,
+                                "textblock": [
+                                    "Um einen Missbrauch zu vermeiden oder zu minimieren, dass sinnlose Warnungen getriggert werden, die gar nicht auf realen Risikokontakten beruhen, wurde entscheiden, dass ein Anwender mit der neuen Funktion nur einmal innerhalb einer bestimmten Zeitspanne Andere warnen kann.",
+                                    "Nach Ablauf dieser Zeitspanne ist es dann wieder möglich, auf Basis eines positiven Tests in der App zu warnen.",
+                                    "Wir empfehlen aber weiterhin, sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig zu verhalten, dass keine anderen Personen angesteckt werden."

```

Comment:
The question, which is in the title of this FAQ entry ("Soll ich nochmal warnen?"), wasn't actually answered at all. I suggest rewriting this article like this:

```suggestion
                                "title": "Einige Tage nach dem letzten positiven Test bin ich beim Selbsttest noch immer positiv. Soll ich nochmal warnen?",
                                "anchor": "warn_without_tan_still_positive",
                                "active": false,
                                "textblock": [
                                    "Da wir weiterhin empfehlen, sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig zu verhalten, dass eine Ansteckung anderer nicht möglich ist. Wenn Sie sich so verhalten, ist eine weitere Warnung nicht notwendig.",
                                    "Beachten Sie bitte außerdem, dass um eine missbräuchliche Nutzung der Funktion zum sinnlosen Auslösen von Warnungen, die gar nicht auf realen Risikokontakten beruhen, zu vermeiden, entschieden wurde, dass ein Anwender mit der neuen Funktion nur einmal innerhalb einer bestimmten Zeitspanne Andere warnen kann.",
                                    "Nach Ablauf dieser Zeitspanne ist es dann wieder möglich, auf Basis eines Tests, der ohne TAN in der App registriert wurde, zu warnen."
```

Reply:
Changed to 
![image](https://user-images.githubusercontent.com/88365935/212058475-383a1274-563e-478f-9ca5-67b2235a4fce.png)
![image](https://user-images.githubusercontent.com/88365935/212058478-1f46db3d-09cc-4155-94ab-9014c4afdbe9.png)

Reply:
Looks good to me @larswmh, conversation can be resolved!